src/models/DotAppearance.ts

```diff
@@ -0,0 +1,22 @@
+import Serializable from "./util/Serializable";
+/**
+ * Defines appearance of a dot
+ *
+ * @property filled         - Wether the dot is filled
+ * @property fill           - The fill color of a dot
+ * @property color          - The border color of a dot and its slash
+ * @property slashed        - Wether the dot has a slash
+ * @property angle          - The angle of the slash
+ */
+export default class DotAppearance extends Serializable<DotAppearance> {
```

Comment:
Here's an alternative idea... Let's not store `DotAppearance` with `Show`. Let's make the dot appearances the same for every show. For example, `dotTypeIndex = 0` is always an open dot, `dotTypeIndex = 1` is always a dot with a forward slash, etc. We can make an array in `models/util/constants` for dot appearance data.

Reply:
ping on this ^

Something like

```ts
export interface DotApperance {
  filled: boolean;
  // ...

export const DOT_APPEARANCES: DotAppearance[] = [
  {
    filled: true,
    // ...
  },
  {
    filled: false,
    // ...
  }
]
```

Which is stored in `models/util/constants`.

Or do you think there's some value in being able to customize the dot type appearances for a specific show?

Reply:
I was thinking keeping the dot appearances separate for different stunsheets for a couple of reasons. 
1: The charters could make a group all have the same appearance for different stuntsheets. For example, if there is a group of people over multiple stuntsheets that stand still, instead of having them get different dot appearances depending on when they're charted we can just have them have the same appearance
2: In the future we can allow funky dot appearances (like letters or emojis or images...)

Reply:
Although now that I think about it, when importing shows from older clacharts we would have to have an appearance associated with each type....

Reply:
Agreed that import/export will be more straightforward with a constant set of appearances. I believe both CalChart and the CalChart Viewer operate in this way right now (far CalChart Viewer, see [https://github.com/calband/calchart-viewer/blob/master/other/file-formats/viewer.json#L33](viewer file specification).

In the long-term, I agree it would be cool to allow the user to create more custom shapes. When that day comes, it may still be useful to have some 'default set of shapes' in a constant somewhere that are available out-of-the-box when a charter starts a new show; so maybe this constant set of dot appearances will continue to be useful when there is support for more funky appearances.

Reply:
Yeah, makes sense! I'm fine either way to get this PR merged.

Reply:
Perhaps we should have that be something we can do later.